tests/integration/upsertGraph.js

```diff
@@ -969,7 +969,40 @@ module.exports = (session) => {
             model2Relation1: []
           }]
         });
-      });
+      })
+    });
+
+    it('should insert root model with an id instead of throwing an error if `insertMissing` option is true', () => {
+      const upsert = {
+        // This doesn't exist.
+        id: 3,
+        model1Prop1: 'updated root 3',
```

Comment:
Actually model with id 3 does exist. See the inserted object in `beforeEach`. Use some ridiculous number like 1234567 or the `NONEXISTENT_ID` constant instead.